_print_char.c

```diff
@@ -0,0 +1,12 @@
+#include "main.h"
+
+/**
+* _print_char - prints a charachter
+* @c:  the character to print
+* Return: 1 (success)
+ */
+
+int _print_char(char c)
```

Comment:
I think we should pass the va_list through the function not the character itself and function should access the character
https://en.cppreference.com/w/c/variadic/va_list/
also check the file [ `_print_str.c` ](https://github.com/Antonious-Awad/printf/blob/main/_print_str.c)